### DISCUSSION TITLE
Can it preview the content of the folder instead of just the details (data/size) from outside?

### DISCUSSION THREAD

**Discussion by Tex91** [QL-Win/QuickLook#1347] [Ideas]
I just discovered this and it's absolutely amazing. I was a Mac user for a long time and I got used to living without it for a while after switching to Windows but this brings everything back, fantastic work. I thought it would be really cool if there was an option to preview the contents of the folder (perhaps giving the ability to drag and drop files for example) with QuickLook rather than just see the details from outside? English isn't my first language but I hope was able to explain my question clearly. Thanks!

**Comment by rabelux** [comment]
Check out the FolderViewer plugin listed here: https://github.com/QL-Win/QuickLook/wiki/Available-Plugins
There is no interactivity but it should provide the preview you're looking for.

**Reply by Tex91** [reply]
Awesome! It worked. Thanks a lot.
